File: src/main/java/com/vjiki/music/service/SongLikeService.java
package com.vjiki.music.service;

import java.util.UUID;

import com.vjiki.music.dto.SongLikeResponse;

public interface SongLikeService {

    void likeSong(UUID userId, UUID songId);

    void dislikeSong(UUID userId, UUID songId);

    SongLikeResponse getLikeDislikeInfo(UUID userId, UUID songId);
}

File: bin/main/com/vjiki/music/service/SongLikeService.kt
package com.vjiki.music.service

import com.vjiki.music.dto.SongLikeResponse
import java.util.UUID

interface SongLikeService {
    fun likeSong(userId: UUID, songId: UUID)
    fun dislikeSong(userId: UUID, songId: UUID)
    fun getLikeDislikeInfo(userId: UUID, songId: UUID): SongLikeResponse
}

